File: app/app/src/main/java/io/github/homework/database/MyDatabaseHelper.java
package io.github.homework.database;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;
import android.util.Log;

import androidx.annotation.Nullable;

import io.github.homework.MainActivity;

public class MyDatabaseHelper extends SQLiteOpenHelper {
    public MyDatabaseHelper(@Nullable Context context, @Nullable String name, @Nullable SQLiteDatabase.CursorFactory factory, int version) {
        super(context, name, factory, version);
    }

    private String createMusic ="create table Music(" +
            "    id integer primary key autoincrement," +
            "    name text," +
            "    author text," +
            "    cover integer," +
            "    resrouce integer" +
            ")";



    @Override
    public void onCreate(SQLiteDatabase db) {
        db.execSQL(createMusic);
        Log.d("==>","创建数据库成功");
    }

    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {

    }
}

File: storagetest/app/src/main/java/io/github/storagetest/MyDatabaseHelper.kt
package io.github.storagetest

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper
import android.widget.Toast

/**

 * @Auther: fushaolei

 * @datetime: 2021/5/9

 * @desc:

 */
class MyDatabaseHelper(val context: Context, name: String, version: Int) :
    SQLiteOpenHelper(context, name, null, version) {
    private val createBook = """
        create table Book (
        id integer primary key autoincrement,
        author text,
        price real,
        pages integer,
        name text
        )
    """.trimIndent()

    override fun onCreate(db: SQLiteDatabase?) {
        db?.execSQL(createBook)
        Toast.makeText(context, "Create Table Succeeded", Toast.LENGTH_SHORT).show()
    }

    override fun onUpgrade(db: SQLiteDatabase?, oldVersion: Int, newVersion: Int) {
        TODO("Not yet implemented")
    }
}